client/src/js/app.js

```diff
@@ -979,15 +983,18 @@ function localeConfig(tmhDynamicLocaleProvider) {
 }
 
 // redirect to login if not signed in.
-function startupConfig($rootScope, $state, SessionService, amMoment, Notify) {
+function startupConfig($rootScope, $state, SessionService, amMoment, Notify, $location) {
 
   // make sure the user is logged in and allowed to access states when
   // navigating by URL.  This is pure an authentication issue.
   $rootScope.$on('$locationChangeStart', function (event, next) {
-
     var isLoggedIn = !!SessionService.user;
     var isLoginState = next.indexOf('#/login') !== -1;
 
+    if($location.$$path === '/error403'){
```

Comment:
The next $location URL is in `next`.  Instead of importing `$location`, you can just do this:
```js
if (next.indexOf('/error404') !== -1) {
  $state.go('/error403');
}
```

It might even be possible to simply call `return;` instead of `$state.go('/error403')`.  That would be even cleaner, since a `$stateChange` event will not be fired twice.

[Relevant documentation](https://docs.angularjs.org/api/ng/service/$location#$locationChangeStart)